example.py

```diff
@@ -51,9 +59,11 @@ def gen_mon(client, device_id):
                 if data:
                     try:
                         res = model.decode_monitor(data)
+                        print(res)
                     except ValueError:
                         print('status data: {!r}'.format(data))
-                    else:
+                """
+                else:
```

Comment:
Seems like we'll want to bring this back at some point.

Reply:
Yeah. It's commented out for now because the v1/v2 api produced very different data.